src/ResourceManager/Version2016_09_01/AzureRMCmdlet.cs

```diff
@@ -37,7 +39,7 @@ namespace Microsoft.Azure.Commands.ResourceManager.Common
     /// <summary>
     /// Represents base class for Resource Manager cmdlets
     /// </summary>
-    public abstract class AzureRMCmdlet : AzurePSCmdlet
+    public abstract class AzureRMCmdlet : AzurePSCmdlet, IDynamicParameters
```

Comment:
Does it affect the cmdlets that have already implemented `IDynamicParameters`?

Reply:
I tested with `Add-AzIotHubModule -AuthMethod x509_thumbprint -PrimaryThumbprint` and it worked well on Windows PowerShell and powershell 7.
Supposedly the reason there's no error about conflicting methods is because [CS0109](https://docs.microsoft.com/en-us/dotnet/csharp/misc/cs0109) is a **compiler warning**.